src/scsiencrypt.cpp

```diff
@@ -369,17 +373,19 @@ bool SCSIExecute(std::string tapedrive, unsigned char *cmd_p, int cmd_len,
 #error "OS type is not set"
 #endif
 #ifdef DEBUGSCSI
-  std::cout << "SCSI Command: ";
+  std::cerr << "SCSI Command: ";
```

Comment:
Why should this go to `cerr` instead of cout?

Reply:
It's diagnostic and should go to stderr. I'd actually say most of the print statements in this program are status messages that are not "output" and should go to stderr.

Reply:
Some users have written script that rely on the output going to stdout.  This would be breaking changes for them.  I think I would prefer that any breaking changes go in a separate PR all together.

Reply:
To clarify, I think these changes are the right move, but I would prefer them in a new minor (or major) version